### PR TITLE
Include Windows in Travis CI build matrix

### PR DESCRIPTION
**Pull request by ardovm**
This is an attempt to solve issue #105

### CODE REVIEW

**Comment by ikus060** [comment]
Really good job @ardovm
It one step forward to have an automated CICD pipeline. As recommended, I propose to get this merge into master and breakdown the remaining work to have a fully working CICD pipeline: test, pyinstaller, etc.

File: .travis.yml [hunk 1]
```diff
@@ -1,13 +1,44 @@
 language: python
 
+os: linux
 dist: bionic
 
+install: pip3 install --upgrade pip tox pyxattr pylibacl setuptools-scm PyInstaller
```
Comment (ericzolf) [inline .travis.yml]: I think the PyInstaller is here too much as the line is only for Linux.

**Comment by ericzolf** [comment]
Thanks a lot, especially the static compilation is a trait of genius and will make deployment of the wheel a lot easier!

**Comment by ardovm** [comment]
Thank you for integrating this! I am very happy I can contribute to this project.

**Pull request merged**